Formula/ivtools.rb

```diff
@@ -0,0 +1,22 @@
+class Ivtools < Formula
+  desc "X11 vector graphic servers"
+  homepage "http://www.ivtools.org"
+  url "https://github.com/vectaport/ivtools/archive/refs/tags/ivtools-2.0.11d.tar.gz"
+  sha256 "8c6fe536dff923f7819b4210a706f0abe721e13db8a844395048ded484fb2437"
+  license "MIT"
+
+  depends_on "ace"
+  depends_on "libx11"
+  depends_on "libxext"
+
+  def install
+    cp "Makefile.orig", "Makefile"
+    system "./configure.hd", prefix.to_s, man.to_s
```

Comment:
```suggestion
    ace = Formula["ace"]
    args = %W[--mandir=#{mandir} --with-ace=#{ace.opt_include} --with-ace-libs=#{ace.opt_lib}]
    system "./configure", *std_configure_args, *args
```
Let's not hardcode `/usr/local` here, since that's not always where Homebrew is installed.